assets/pages/oneoff-contributions/components/oneoffContributionsPayment.jsx

```diff
@@ -123,6 +125,7 @@ function OneoffContributionsPayment(props: PropTypes, context) {
 function mapStateToProps(state) {
   return {
     isTestUser: state.page.user.isTestUser || false,
+    isPostDeploymentTestUser: state.page.user.isPostDeploymentTestUser || false,
```

Comment:
Since `isPostDeploymentTestUser` has type boolean. It will be always defined, therefore `|| false` it is not necessary since it will be never be used.

Reply:
Hmm, I was thinking that. Presumably @jacobwinch you've followed the pattern used for `isTestUser`? Is there a reason that `isTestUser` is a Maybe boolean in the first place? Can't it also be a straight boolean?

Reply:
Good point - I'll have a look at that in a separate PR (I can't think of a good reason why it shouldn't be a straight boolean either...)